CMakeLists.txt

```diff
@@ -85,6 +85,26 @@ endif(WITH_CCACHE)
 
 option(WITH_SCCACHE "Build with sccache.")
 if(WITH_SCCACHE)
+  if(NOT FORCE_MAX_JOBS)
+    execute_process(
+      COMMAND "sccache" "--dist-status"
```

Comment:
would be better if we coud check for the existence of `sccache` before using it, like
```cmake
find_program(SCCACHE_EXECUTABLE sccache)
if(NOT SCCACHE_EXECUTABLE)
  message(FATAL_ERROR "Can't find sccache.")
endif()
```
if we bump up the minimal required version of cmake to 3.18 and up, we could instead use
```cmake
find_program(
  SCCACHE_EXECUTABLE
  sccache
  REQUIRED)
```

Reply:
Great point; I moved the check to run before we try to invoke sccache.